src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -810,8 +813,10 @@ const ComposeForm = props => {
 };
 
 ComposeForm.propTypes = {
+  categories: PropTypes.array,
```

Comment:
Does categories need to be included in the propTypes?

Reply:
you're right, this gotta go away since I am removing it is a prop. Will push a correction